vignettes/spatialHotspot.Rmd

```diff
@@ -0,0 +1,48 @@
+---
+title: "Vision Hotspot Vignette"
+author: "Yanay Rosen"
+date: "9/30/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+devtools::load_all("~/Desktop/VISION/") # change to just load VISION
+library(reticulate)
+```
+
+## Creating a Vision Object
+
+First, we create a Vision object
+```{r create}
+# Read the expression and meta data
+expr <- read.table("VisHotspotVignetteFiles/expr.tsv", check.names = FALSE, sep = "\t")
+meta <- read.table("VisHotspotVignetteFiles/meta.tsv", check.names = FALSE, sep = "\t", row.names = 1, header=TRUE)
+
+# Signature file
+sig = "VisHotspotVignetteFiles/h.all.v5.2.symbols.gmt"
+
+# Read and create the coordinates
+pos <- read.table("VisHotspotVignetteFiles/BeadLocationsForR.csv", sep=",", check.names = FALSE, row.names=1, header=TRUE)
+pos["X"] <- pos$ycoord
+pos["Y"] <- -1 * pos$xcoord
+pos <- pos[c("X", "Y")]
+
+# Construct the Vision object
+vis <- Vision(expr, signatures=c(sig), latentSpace = pos, meta=meta) # TODO add relevant signatures
+```
+
+Next, we can perform the normal Vision analysis using the tree as the latent space. We need to tell Vision to use the Tree as the latent space and to calculate neighbors. 
```

Comment:
We're not using the tree as the latent space here - can you be more specific about what this is doing with the spatial barcodes?

Let's be careful to make it clear that you don't need to run Hotspot on spatial data, and that we have applications on phylogenies & RNA-seq datasets.

Reply:
Fixed in latest commit!